packages/apputils-extension/style/contextualhelp.css

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) Jupyter Development Team.
+ * Distributed under the terms of the Modified BSD License.
+ */
+:root {
+  --jp-private-shortcuts-table-padding-vertical: 9px;
+  --jp-private-shortcuts-table-padding-right: 14px;
+  --jp-private-shortcuts-key-padding-horizontal: 5px;
+  --jp-private-shortcuts-key-padding-vertical: 3px;
+}
+
+tr,
+td {
```

Comment:
Please use selectors specific to the shortcuts table.

Reply:
@krassowski Thanks for the comments. My knowledge of css is not very high. I tried to remove this but the reason I used directly tr and td is that it was the only way I found to manage to apply the padding that were suggested in the issue from a design point of view.